dspace/config/registries/process-types.xml

```diff
@@ -0,0 +1,20 @@
+<dspace-dc-types>
+
+    <dspace-header>
+        <title>DSpace Process Types</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>process</name>
+        <namespace>http://dspace.org/process</namespace>
+    </dc-schema>
+
+    <dc-type>
+        <schema>process</schema>
+        <element>type</element>
+        <qualifier></qualifier>
+        <scope_note></scope_note>
```

Comment:
This should have a scope note, as its purpose is unclear...and it sounds like we do *not* want people to use this field. I'm also not sure I like having a schema with a *single* metadata field.  We should think about creating a `dspace` schema for fields that are internal, rather than creating new schemas every time we need an internal field.

Reply:
Again, please resolve prior feedback.

Reply:
The metadata field was moved to a new "dspace" schema. We need to store the file type because we need to group files by type in our rest output, can be seen here: https://github.com/DSpace/Rest7Contract/blob/master/processes-endpoint.md#execution-file-output-list